avogadro/io/cjsonformat.cpp

```diff
@@ -380,6 +380,20 @@ bool CjsonFormat::write(std::ostream& file, const Molecule& molecule)
     unitCell["beta"] = molecule.unitCell()->beta() * RAD_TO_DEG;
     unitCell["gamma"] = molecule.unitCell()->gamma() * RAD_TO_DEG;
     root["unit cell"] = unitCell;
+
+    json vectors;
+    vectors.push_back(molecule.unitCell()->aVector().x());
+    vectors.push_back(molecule.unitCell()->aVector().y());
+    vectors.push_back(molecule.unitCell()->aVector().z());
+    
+    vectors.push_back(molecule.unitCell()->bVector().x());
+    vectors.push_back(molecule.unitCell()->bVector().y());
+    vectors.push_back(molecule.unitCell()->bVector().z());
+
+    vectors.push_back(molecule.unitCell()->cVector().x());
+    vectors.push_back(molecule.unitCell()->cVector().y());
+    vectors.push_back(molecule.unitCell()->cVector().z());
+    root["cell vectors"] = vectors;
```

Comment:
I have been using camel case for new keys, they are easier to use in all languages, i.e. cellVectors, I was hoping avoid any new keys with spaces in them.

Reply:
I’m coming back to this now. I’m happy to use cellVectors as the key. I’m not sure what you mean by “vectors” for the storage? Do you want to store them as [ [ax, ay, az], [bx, by, bz], [cx, cy, cz] ] instead? That’s fine, but it’s different from how we store atomic coordinates - thus I went with an array of 9 values.

Reply:
I meant the key name, that was all. I actually want to use this in our server platform now too. I was suggesting storing the whole thing in the "unit cell" key as "vectors" as the "unit cell" already makes it clear that it is defining the molecule's unit cell. I think the way you stored it makes sense.

Reply:
Okay, I’ll get that done later this afternoon, thanks.